src/components/Articles/scss/mystile.scss

```diff
@@ -0,0 +1,99 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
```

Comment:
Please read this article of the next documentation to use to optimize fonts for next 13 https://nextjs.org/docs/pages/building-your-application/optimizing/fonts